src/apps/search-result/search-result-skeleton.tsx

```diff
@@ -7,19 +7,23 @@ import CardListItemSkeleton from "../../components/card-item-list/card-list-item
 export interface SearchResultSkeletonProps {
   q: string;
   dataCy?: string;
+  isAdvancedSearch?: boolean;
```

Comment:
This is just a consideration:
Instead of making this aware of the concept: "advanced search" you can also call the prop: `hideFacetLine`.
In that way it does not need to be modified if the "advanced search" (god forbid that) would be removed in the future.

Reply:
That's a good idea, actually :)